### ISSUE TITLE
Fuel Consumption in the "R"-Menu

### ISSUE THREAD

**Issue by xNccPlay** [EternalBlueFlame/Traincraft-5#419]
Hey guys, today I noticed, that all trains, even electric and steam trains are showing behind "Fuel Consumption" X mB/t instead of Redstone or Coal or anything else. The ICB says, that he has no Fuel Comsumption. TC Version 017

greetings, xNccPlay

![2018-08-22_15 02 16](https://user-images.githubusercontent.com/1348874/44465170-15837800-a61d-11e8-88c0-80eadd9b03b1.png)
![2018-08-22_15 04 28](https://user-images.githubusercontent.com/1348874/44465171-15837800-a61d-11e8-9e67-eac33b27e38d.png)
![2018-08-22_15 04 39](https://user-images.githubusercontent.com/1348874/44465172-161c0e80-a61d-11e8-92c7-64b41de487d4.png)

**Comment by EternalBlueFlame** [comment]
In the fuel consumption is an ascetic for displaying one of the necessary enum values, however it's not actually used for diesel or electric trains as they don't use the old fuel variables (talk about a waste of system resources).
It's an oversight from at least the 1.6.2 era of the mod, if not longer ago than that.

**Comment by xNccPlay** [comment]
I remembered, that sometimes earlier there stands something like "4 every 10 seconds" in electric trains, same for steam trains...And that must have worked on 014... 015&016 I'm not sure about..

**Comment by EternalBlueFlame** [comment]
Yeah the wording was changed in one of our more "recent" revisions, but the actual use methods are still the same.
